middleware/TelemetryHandler.md

```diff
@@ -15,6 +15,7 @@ Provide a mandatory middleware component that attaches metadata to a Graph reque
     | **Core** | `graph-{lang}-core/{major}.{minor}.{build}` | `graph-dotnet-core/1.16.0` |
     | **v1.0 Service Model** | `graph-{lang}/{major}.{minor}.{build}` | `graph-dotnet/1.16.0` |
     | **Beta Service Model** | `graph-{lang}-beta/{major}.{minor}.{build}` | `graph-dotnet-beta/1.16.0` |
+    | **Future Service Model** | `graph-{lang}-{endpoint}/{major}.{minor}.{build}` | `graph-dotnet-v2.0/1.16.0` |
```

Comment:
if we go with a grand fathering approach, what does that mean for new sdks? (thinking about go here for example)

Reply:
It would mean that service libraries for new sdks that target v1.0 would use the existing standard.

Reply:
so v1.0 would stay without version information, even for new sdks. Ok let's table that until we have a resolution on #63